app/lib/forms/itt_provider.rb

```diff
@@ -0,0 +1,33 @@
+module Forms
+  class IttProvider < Base
+    QUESTION_NAME = :itt_provider
+
+    attr_accessor QUESTION_NAME
+
+    validates QUESTION_NAME, presence: true
+
+    def self.permitted_params
+      [QUESTION_NAME]
+    end
+
+    def question
+      Forms::QuestionTypes::AutoCompleteIttProvider.new(name: QUESTION_NAME)
+    end
+
+    def next_step
+      if approved_itt_provider?
+        :choose_your_npq
+      end
```

Comment:
Where would it go otherwise?

Reply:
ahh yes i need to confirm this but the flow diagram didnt have any other option?

Reply:
Will definitely need to figure that out, I think if you don't provide a fallback it'll trigger a 500. Unless it's meant to just raise an error in which case `approved_itt_provider?` should go in the validations on the form and then :choose_your_npq is always the next step if validations pass.